Issutracker : #i49682#
CWS         : N/A
Author      : NAKATA Maho <maho@openoffice.org> (JCA)
Description : correct version number of resultant shared lib

Index: xmlsec1-1.2.6.patch
===================================================================
RCS file: /cvs/external/libxmlsec/xmlsec1-1.2.6.patch,v
retrieving revision 1.5
diff -u -r1.5 xmlsec1-1.2.6.patch
--- libxmlsec/xmlsec1-1.2.6.patch	18 May 2005 10:11:24 -0000	1.5
+++ libxmlsec/xmlsec1-1.2.6.patch	22 May 2005 01:43:49 -0000
@@ -756,6 +756,35 @@
 *** misc/xmlsec1-1.2.6/ltmain.sh	2004-08-26 08:00:15.000000000 +0200
 --- misc/build/xmlsec1-1.2.6/ltmain.sh	2005-05-09 19:55:51.745319024 +0200
 ***************
+*** 2994,3006 ****
+  	  ;;
+  
+  	freebsd-aout)
+! 	  major=".$current"
+! 	  versuffix=".$current.$revision";
+  	  ;;
+  
+  	freebsd-elf)
+! 	  major=".$current"
+! 	  versuffix=".$current";
+  	  ;;
+  
+  	irix | nonstopux)
+--- 2994,3006 ----
+  	  ;;
+  
+  	freebsd-aout)
+! 	  major=.`expr $current - $age`
+! 	  versuffix="$major.$age.$revision"
+  	  ;;
+  
+  	freebsd-elf)
+! 	  major=.`expr $current - $age`
+! 	  versuffix="$major.$age.$revision"
+  	  ;;
+  
+  	irix | nonstopux)
+***************
 *** 3564,3570 ****
   		fi
   	      else
